project/cookiecutter.json

```
@@ -85,6 +87,12 @@
       "docker_hub": "Docker Hub",
       "gitlab": "GitLab"
     },
+    "storage": {
+      "__prompt__": "What storage backend should we use in the stack?",
```

The more common term I've seen used is "persistent storage". "storage backend" is strange, as if storage _is_ the backend, not just part of the backend.

```suggestion
      "__prompt__": "Which persistent storage do you want to use in the stack?",
```

---

I've changed the terminology to "persistent storage" now. Also, I agree that "we" sounds a bit strange, but it is what is used by some of the other prompts while "you" isn't being used anywhere right now. I changed it to just "which... to use".

---

That's fine.

As a side note, in general, documentation is moving toward directly addressing the reader or user. For Plone Documentation, we've adopted the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/) as a guide, and somewhat enforced through [Vale](https://6.docs.plone.org/contributing/documentation/authors.html#authors-english-label). Documentation and UI/UX overlap in some regards, so that's where I get style suggestions.

Plone lacks an official UI/UX style guide. I think it would be good to adopt an existing guide from a larger organization. But that's outside the scope of this PR.